pkg/validate/security_context.go

```diff
@@ -402,6 +424,41 @@ var _ = framework.KubeDescribe("Security Context", func() {
 
 			checkNetworkManagement(rc, containerID, false)
 		})
+	})
+
+	Context("runtime should support seccomp security context", func() {
```

Comment:
nits: `[security context]` same with other places.

Reply:
the top level bucket is "[k8s.io] security context" 
This context is for seccomp tests within [k8s.io] security context.

Not sure what the point is of having brackets to both the left and right most extent of the individual tests.

Let me refactor all of the titles making it easier to regexp and only having brackets on the left for ease of reading.

Reply:
@feiskyer On second thought, I also think it is fine to remove the tag given how cri validation test is organized today. :)